xtask/migrations/20230620132156_benchmarks.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE Host(
+    id SERIAL,
+    os TEXT NOT NULL,
+    cpu TEXT NOT NULL,
+    memory BIGINT UNSIGNED NOT NULL,
+    CONSTRAINT Reporter_pk PRIMARY KEY (id)
+);
+
+CREATE TABLE Reporter(
```

Comment:
Nitpick of mine, but can we rename `Reporter` to `Report` and `Metrics` to `Metric`?